dll/win32/shell32/dialogs/recycler_prop.cpp

```diff
@@ -325,7 +331,26 @@ RecycleBinDlg(
                     {
                         uResult = GetDlgItemInt(hwndDlg, 14002, &bSuccess, FALSE);
                         if (bSuccess)
-                            pItem->dwMaxCapacity = uResult;
+                        {
+                            /* Obtain the information of available disk size */
+                            GetDiskFreeSpaceExW(NULL, NULL, NULL, &FreeBytesSpace);
```

Comment:
Shouldn't be GetDiskFreeSpaceExW first parameter not NULL? if for example somebody would want to select size for other drives other than root (C:)?